WelsonJS.Toolkit/WelsonJS.Launcher/ResourceTools/Completion.cs

```diff
@@ -175,7 +178,7 @@ private void SearchAllExecutables(string path, SearchOption searchOption = Searc
             }
             catch (Exception ex)
             {
-                Trace.TraceInformation("Error enumerating executables in '{0}': {1}", path, ex.Message);
+                _logger.Info("Error enumerating executables in '{0}': {1}", path, ex.Message);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Input is being passed directly to log statements, which could allow attackers to inject malicious content into log files. This can lead to log forging, data integrity issues, or even remote code execution in some cases. Sanitize all external input before logging, especially data from HTTP requests. Use encoding functions like HtmlEncode() or validated libraries to properly sanitize the input. Learn more - https://cwe.mitre.org/data/definitions/117.html